src/SearchableExtension.php

```diff
@@ -19,6 +41,52 @@ class SearchableExtension extends DataExtension
     "getSearchableTitle" => "Text",
     "getSearchableSummary" => 'HTMLText',
   ];
+
+  /**
+   * updateSettingsFields
+   * Adds the SearchTerms GridField to the settings tab
+   * This should only be applied to SiteTree objects
+   *
+   * @param FieldList $fields
+   * @return void
+   **/
+  public function updateSettingsFields(FieldList $fields)
```

Comment:
This being under settings makes sense for pages, but for other searchable dataobjects, like articles and content layouts, they do not have settings fields.

Reply:
@tiller1010 Take a look at this again, I updated so the search terms are added to all objects

Reply:
I forgot to mention the content layout, blog, etc....need their getIndexQuery functions updated to include search terms. I figure I would do that after this is merged and released